admin/partials/doppler-for-woocommerce-settings.php

```diff
@@ -62,13 +62,21 @@
 
         default:
             if( isset($_POST['dplr_subscribers_list']) && $this->validate_subscribers_list($_POST['dplr_subscribers_list']) && current_user_can('manage_options') && check_admin_referer('map-lists') ){
-                update_option( 'dplr_subscribers_list', $this->sanitize_subscribers_list($_POST['dplr_subscribers_list']) );
+
+                $sanitized_subscribers_list = $this->sanitize_subscribers_list($_POST['dplr_subscribers_list']);
+
+                update_option( 'dplr_subscribers_list', $sanitized_subscribers_list);
                 $this->set_success_message(__('Subscribers lists saved succesfully', 'doppler-for-woocommerce'));
+                
                 $this->reset_buyers_and_contacts_last_synch();
+
+                $subscribers_lists = $sanitized_subscribers_list;

```

Comment:
Que les parece @acatacoravns, @blancfabian ? 
```suggestion
                $subscribers_lists = $this->sanitize_subscribers_list($_POST['dplr_subscribers_list']);

                update_option( 'dplr_subscribers_list', $subscribers_lists);
                $this->set_success_message(__('Subscribers lists saved succesfully', 'doppler-for-woocommerce'));
                
                $this->reset_buyers_and_contacts_last_synch();

```